.github/workflows/ci.yml

```diff
@@ -8,19 +8,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 15.x]
+        node-version: [12.x, 14.x, 16.x]
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
       - name: Cache node_modules
         id: cache-modules
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
           path: node_modules
           key: ${{ matrix.node-version }}-${{ runner.OS }}-build-${{ hashFiles('yarn.lock') }}
       - run: yarn install
       - run: yarn test
       - run: yarn build
       - run: yarn lint
+      - run: yarn prettier-check
```

Comment:
not sure we need this

Reply:
True. lint-staged should take care of it.